### ISSUE TITLE
New v5.0.0 version is always rebasing urls

### ISSUE THREAD

**Issue by eneko89** [jakubpawlowicz/clean-css-cli#54]
Urls are getting rebased always, even when `--with-rebase` option is not used.

Both of these rebase urls and produce the same results:
```
cleancss assets/css/styles.css -o dist/css/styles.min.css
cleancss assets/css/styles.css -o dist/css/styles.min.css --with-rebase
```

In the source css, I have font and img references like these:
```
url('../fonts/roboto-400-latin.woff2')
url('../img/bg.jpg')
```

And with v5.0.0, I get these urls in the minified file, both with and without `--with-rebase`:
```
url('../../assets/fonts/roboto-400-latin.woff2')
url('../../assets/img/bg.jpg')
```

With previous v4.3.0, I used `--skip-rebase` and was getting:
```
url('../fonts/roboto-400-latin.woff2')
url('../img/bg.jpg')
```

I'm running macOS Big Sur 11.2.

**Comment by jakubpawlowicz** [comment]
Hey @eneko89 - can you please provide more details, as described in https://github.com/jakubpawlowicz/clean-css-cli/blob/master/CONTRIBUTING.md ? Thanks!

**Comment by eneko89** [comment]
Sure, I'll edit.

**Comment by eneko89** [comment]
Done @jakubpawlowicz.

Btw, thanks for your quick response and sorry for the inconvenience 🙂

**Comment by jakubpawlowicz** [comment]
No problem @eneko89 Let me have a look at it!

**Comment by jakubpawlowicz** [comment]
It's fixed in 5.0.1 on npm. Cheers!

**Comment by eneko89** [comment]
I can vouch for that. Thanks! 🍻
